test/selenium/specs/ts_all_specs.rb

```diff
@@ -4,4 +4,7 @@
 require_relative "tc_show_injected"
 require_relative "tc_extension_options"
 require_relative "tc_history"
+if @browser == "firefox"
+  require_relative "tc_posting_process"
```

Comment:
You should add comments to the files indicating why these tests are only run for Firefox.